packages/turf-constrained-tin/test.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
+const path = require('path');
+const test = require('tape');
+const tin = require('./').default;
+
+const points = require(path.join(__dirname,  'test', 'Points.json'));
```

Comment:
I'd probably add few more tests, for example with some edge cases to can think of

Reply:
Add 10 more tests including error case